gateway/gateway.go

```diff
@@ -38,6 +40,23 @@ var defaultQueryFrontendAPIs = []string{
 	"/api/prom/api/v1/status/buildinfo",
 }
 
+var defaultAlertManagerAPIs = []string{
+	"/alertmanager",
```

Comment:
```suggestion
	"/alertmanager/",
```
Need everything that is inside /alertmanager/